validator/sawtooth_validator/database/lmdb_nolock_database.py

```diff
@@ -54,6 +54,7 @@ def __init__(self, filename, flag):
                                       create=create,
                                       lock=True)
 
+    # pylint: disable=no-value-for-parameter
```

Comment:
I guess you could try `return txn.stat(self._lmdb)['entries']` to see if it works.